user_docs/en/userGuide.t2t

```diff
@@ -519,6 +518,20 @@ When the menu comes up, You can use the arrow keys to navigate the menu, and the
 | Report clipboard text | NVDA+c | Reports the Text on the clipboard if there is any. |
 %kc:endInclude
 
+++ Speech modes ++[SpeechModes]
+
+There are four speech modes available.
+* Talk: NVDA will just speak normally. This is the default speech mode.
+* On-demand: NVDA will only speak when calling commands whose goal is to speak an information (e.g. report the title of the window); but it will not speak in reaction to actions such as moving the focus or the cursor.

```

Comment:
Since I suspect you will change this to a hyphenated list, I have used that in the below to make it easy to apply my suggestion if you otherwise like it.

I have removed "an information" because "information" is plural in English. I have replaced "calling" (which is often understood in technical circles but may not be how users think of command use), with a more user friendly term.
Lastly, I have added some further clarification about what this mode does.

```suggestion
- On-demand: NVDA will only speak when you use commands which speak information (e.g. report the title of the window); but it will not speak in reaction to actions such as moving the focus or the cursor; and will not speak any screen changes, notifications, etc.
```

Reply:
I'll take this comment into account. But I want to keep the idea that the goal of the command is to report information. Writing "a command which speaks information" is too broad in my view.